task_allocation/Car car
task_allocation/Task finished_task
std_msgs/String result
---
task_allocation/Task[] task_list